lib/modules/manager/kustomize/extract.spec.ts

```diff
@@ -175,6 +176,24 @@ describe('modules/manager/kustomize/extract', () => {
       });
       expect(pkg).toEqual(sample);
     });
+
+    it('should correctly extract an OCI chart', () => {
+      const sample = {
+        depName: 'redis',
+        packageName: 'registry-1.docker.io/bitnamicharts/redis',
+        currentValue: '18.12.1',
+        registryUrls: ['https://registry-1.docker.io/bitnamicharts'],
```

Comment:
```suggestion
```

Reply:
If I make your suggested change in https://github.com/renovatebot/renovate/pull/27163#discussion_r1484625649, this will become `registryUrls: ['oci://registry-1.docker.io/bitnamicharts']` rather than getting removed. Is that what we want?